apps/prairielearn/src/pages/administratorQuery/administratorQuery.html.ts

```diff
@@ -177,7 +177,7 @@ export function AdministratorQuery({
         <main id="content" class="container-fluid">
           <div class="card mb-4">
             <div class="card-header bg-primary text-white d-flex align-items-center">
-              <span class="font-weight-bold text-monospace">${sqlFilename}</span>
+              <h1 class="h6 font-weight-bold text-monospace mb-0">${sqlFilename}</h1>
```

Comment:
Using the sql filename here for the title. It works but it might be strange reading a filename instead here. Let me know if you want me to use something else more standard like "Admin Query" or something here.

Reply:
This is an admin-only page so this matters much less than user-facing ones, but you might consider changing this to something like `Query: <span class="text-monospace">${sqlFilename}</span>` so that it's at least announced as a query and not just a filename.

Reply:
That sounds good. Updated to this in [32a4353](https://github.com/PrairieLearn/PrairieLearn/pull/10440/commits/32a4353c31f3a6e43f583964f80031e262180873).